src/lib/bedrockYaml.ts

```diff
@@ -105,6 +105,53 @@ export const addNewService = (
   fs.writeFileSync(path.join(absPath, YAML_NAME), asYaml);
 };
 
+/**
+ * Saves the bedrock.yaml file
+ * @param dir @param dir Directory where <code>bedrock.yaml</code> file resides.
+ * @param data The data for bedrock.yaml
+ */
+export const save = (dir: string, data: IBedrockFile) => {
```

Comment:
Use this function instead: https://github.com/CatalystCode/spk/blob/master/src/lib/bedrockYaml.ts#L26